### DISCUSSION TITLE
Ability to output logs to file respecting TZ

### DISCUSSION THREAD

**Discussion by tiredofit** [stalwartlabs/smtp-server#36] [Q&A]
I am outputting logs to file and using my own logrotation scheme, and have noticed that logss are outputting in UTC. I have tried padding `TZ=America/Vancouver stalwart-mail ...` hoping that it would start to output in my own time zone to no success. Are there plans to introduce this functionality?

**Comment by mdecimus** [comment]
Not currently supported but should be easy to add if you request an enhancement.

**Reply by tiredofit** [reply]
Thanks! I can file an enhancement request.
